audit-log/README.md

```diff
@@ -0,0 +1,151 @@
+--- 
+layout: pattern
+title: Audit Log
+folder: audit-log
+permalink: /patterns/audit-log/ 
+categories:
+- creational 
+  language: en 
+  tags:
+- Security
+- Data Access
+---
+
+## Name / classification
+
+Audit Log.
+
+## Also known as
+
+Audit trail.
+
+## Intent
+
+Have a simple log of changes, intended to be easily written and non-intrusive.
+
+## Explanation
+
+Real-world example
+
+> A customer changes their address, we want an independent record that this change occured, and 
+> when it has occurred.
+
+In plain words
+
+> Whenever a some property changes, log the change in a file.
```

Comment:
Did you mean "Whenever a property changes, log the change in a file."?

Reply:
I did, thank you for catching that.